releases/patch-releases.md

```diff
@@ -57,26 +57,29 @@ releases may also occur in between these.
 
 ### 1.19
 
-Next patch release is **1.19.4**
+Next patch release is **1.19.5**
 
 End of Life for **1.19** is **2021-09-30**
 
 | PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
 |--- |--- |--- |
+| 1.19.5        | 2020-12-11           | 2020-12-1   |
 | 1.19.4        | 2020-11-06           | 2020-11-11  |
 | 1.19.3        | 2020-10-09           | 2020-10-14  |
 | 1.19.2        | 2020-09-11           | 2020-09-16  |
 | 1.19.1        | 2020-09-04           | 2020-09-09  |
 
 ### 1.18
 
-Next patch release is **1.18.11**
+Next patch release is **1.18.13**
 
 End of Life for **1.18** is **2021-04-30**
 
 | PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
 |--- |--- |--- |
-| 1.18.11       | 2020-11-06           | 2020-11-11  |
+| 1.18.13       | 2020-12-11           | 2020-12-16  |
```

Comment:
The date does not seem to be aligned with  v1.19.5, what is the reason for that?

Reply:
ops...

Reply:
done, thanks!

PTAL